### PR TITLE
ES-24031 Sample App iOS working

### PR DESCRIPTION
**Pull request by karmn92**
Added Sample iOS App using the LOTR SDK

### CODE REVIEW

File: iOSSampleAppWithSDK/SampleAPP/Podfile.lock
Comment (pocha) [inline iOSSampleAppWithSDK/SampleAPP/Podfile.lock:54]: Cant we build from master branch itself ?

File: iOSSampleAppWithSDK/SampleAPP/Podfile.lock
Comment (pocha) [inline iOSSampleAppWithSDK/SampleAPP/Podfile.lock:56]: I see the repos below are directly added above. Why they are needed here again ?

File: iOSSampleAppWithSDK/SampleAPP/Pods/Manifest.lock
Comment (pocha) [inline iOSSampleAppWithSDK/SampleAPP/Pods/Manifest.lock:1]: This file seems exact replica of Podfile.lock . Is this really needed ?

**Review by pocha — CHANGES_REQUESTED** [review]
Lets remove the extra `SampleAPP` directory. Lets have all the code inside iOSSampleAppWithSDK directly. 

Too many extra files/directories which are not required like Pods etc. Please remove them.

File: iOSSampleAppWithSDK/SampleAPP/Podfile.lock
Comment (karmn92) [reply iOSSampleAppWithSDK/SampleAPP/Podfile.lock]: i will make this change once the sdk pr is merged

File: iOSSampleAppWithSDK/SampleAPP/Podfile.lock
Comment (karmn92) [reply iOSSampleAppWithSDK/SampleAPP/Podfile.lock]: there are the libraries used by the LocusLotrSDK

File: iOSSampleAppWithSDK/SampleAPP/Pods/Manifest.lock
Comment (karmn92) [reply iOSSampleAppWithSDK/SampleAPP/Pods/Manifest.lock]: this generated by cocoapods this no loger there as have added /pods in .gitignore

**Review by kanj-locus — COMMENTED** [review]
Do the auto-generated project files (in xcodeproj & xcworkspace etc.) contain any information about developer credentials? If some one clones this, they shouldn't be able to see or use locus developer ids. In fact, we should remove all the files that are not required to import the project in xcode.

**Comment by karmn92** [comment]
@kanj-locus there is no such info and even if it is present they will not be able to use the account as they dont have the private key in the keychain

File: iOSSampleAppWithSDK/Podfile.lock
Comment (pocha) [inline iOSSampleAppWithSDK/Podfile.lock:80]: I think we should remove the commit. It will by default pick up the latest commit from the master branch IMO.

**Review by pocha — APPROVED** [review]
I see a lot of items in Podfile.lock but Podfile does not contain them. Is that alright ? 

Approving with minor comments. You dont have to re-request review.

**Review by kanj-locus — APPROVED** [review]

File: iOSSampleAppWithSDK/Podfile.lock
Comment (karmn92) [reply iOSSampleAppWithSDK/Podfile.lock:80]: this happens only when he runs the pod install on the project

**Pull request merged**
